rtlib/generic_opts.hh

```diff
@@ -101,7 +155,45 @@ class Opts {
 #ifdef LIBRNA_RNALIB_H_
         << " (-[tT] [0-9]+)? (-P PARAM-file)?"
 #endif
-        << " (-[drk] [0-9]+)* (INPUT|-f INPUT-file)\n\n"
+        << " (-[drk] [0-9]+)* (-h)? (INPUT|-f INPUT-file)\n"
+        << "--help   ,-h                          print this help message\n"
+#ifdef CHECKPOINTING_INTEGRATED
+        << "--checkpointInterval,-p  d:h:m:s      specify the periodic "
+        << "checkpointing\n"
+        << "                                      interval,default: 0:1:0:0 "
+        << "(1h)\n"
+        << "--checkpointOutput,-O   PATH/PREFIX   set path where to store "
+        << "the checkpoints,\n"
+        << "                                      default: current working "
+        << "directory\n"
+        << "                                      Optional: add custom prefix"
+        << "for generated\n"
+        << "                                      files to PATH (e.g. PATH:\n"
+        << "                                      \"/path/to/dir/file_prefix\""
+        << "\n"
+        << "                                      will set PATH to \"/path/to/"
+        << "dir/\"\n"
+        << "                                      and PREFIX to \"file_prefix\""
+        << ").\n"
+        << "                                      Make sure to add a \"/\" to\n"
+        << "                                      the end of PATH if you don't"
+        << "\n"
+        << "                                      wish to add a custom prefix "
+        << "to the files.\n"
+        << "--checkpointInput,-I    LOGFILE       set the path to the Logfile\n"
+        << "                                      of the checkpoints you wish "
+        << "to load.\n"
+        << "                                      (This file was generated "
+        << "along\n"
+        << "                                      with the checkpoint archives."
+        << "\n"
+        << "                                      If it isn't available\n"
+        << "                                      add the path to each archive "
+        << "to a \n"
+        << "                                      text file and provide the \n"
+        << "                                      path to this file).\n"
```

Comment:
cool to know that this is possible, but should you trust a dump that lacks the logfile at all? Not sure if we want to print this in the help message?!

Reply:
If there is no Logfile, probably not. I just thought of e.g. a case where the user at some point realizes that a long-running program crashed, decided to restart the program at a later date and thus copied the table archives to a different directory but forgot to copy the Logfile, in which case it would be good to have a 2nd method to restart a program from a checkpoint.

Reply:
ok, that will be a very advanced user though :-) But keep as is!